x-pack/solutions/security/plugins/security_solution/public/detections/links.ts

```diff
@@ -21,3 +26,17 @@ export const links: LinkItem = {
     }),
   ],
 };
+
+export const alertSummaryLink: LinkItem = {
+  id: SecurityPageName.alertSummary,
+  path: ALERT_SUMMARY_PATH,
+  title: 'Alert summary',
+  capabilities: [`${SECURITY_FEATURE_ID}.show`],
```

Comment:
@tomsonpl 
As @PhilippeOberti mentioned, there are a couple of issues with the config. Both problems can be solved using `capabilities`. We can create a new capability (`ui` action) linked to the new tier, via _pli_config_, and define it here. This will fix both problems, the global nav and the direct access (the "no privilege" page will show up until we create an upsell).

Reply:
Fixed in: #1d4207b679af1d284ff08634d5a9b3061bee9a4f